packages/core-typings/src/IRoom.ts

```diff
@@ -149,6 +149,12 @@ export enum OmnichannelSourceType {
 	OTHER = 'other', // catch-all source type
 }
 
+export enum RoomVerificationState {
+	off = 'off',
```

Comment:
```suggestion
	unverified = 'unverified',
```
WDYT about renaming this state? Also wouldn't this also have "Verified" state?